apps/web/src/components/form/Field.tsx

```diff
@@ -37,6 +39,7 @@ export const Field: React.FC<FieldProps> = props => {
       <FormikField
         id={name}
         aria-describedby={description ? `${name}-description` : null}
+        data-cy={`${cyData}`}
```

Comment:
Is a template needed here?
```suggestion
        data-cy={cyData}
```

Reply:
I don't think so. cyData should always be a string